docs/DEPLOYMENT.md

```diff
@@ -73,6 +83,7 @@ Execute below command
 ```
 kubectl apply -f 00-init/
```

Comment:
`-n tekton-hub`
add this to all kubectl command

Reply:
Instead can't we add a step at the initial stage to create a namespace `tekton-hub`

Reply:
`-n tekton-hub` this for other resources so that they will be created in `tekton-hub` namespace

Reply:
Yep make sense I thought of creating a ns and making it default